app/viewmodels/journeys/adjustments/AdjustedTaxableProfitSummary.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.journeys.adjustments
+
+import models.common.TaxYear
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.table.Table
+import viewmodels.checkAnswers.{buildTable, buildTableAmountRow}
+
+object AdjustedTaxableProfitSummary {
+
+  // TODO all of the following hardcoded values will be replaced with API data from different places based on the calculations (SASS-9523)
+
+  def buildTable1(taxYear: TaxYear)(implicit messages: Messages): Table = {
```

Comment:
Would rename this to `buildYourAdjustedProfitTable` rather than buildTable1. Same with all buildTables bellow.